.github/release-drafter.yml

```
@@ -0,0 +1,214 @@
+      - '.travis.yml'
+  - label: 'performance'
+    title:
+      - '/^perf(\(.+\))?:/'
+  - label: 'security'
+    title:
+      - '/^security(\(.+\))?:/'
+
+# Template for the release body
+template: |
+  ## 📋 Summary
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)